### DISCUSSION TITLE
The option to toggle/hide ViGEmBus devices

### DISCUSSION THREAD

**Discussion by Tvviztid** [LizardByte/community#310] [Sunshine] [answered]
Would love to have the option to prevent virtual gamepads from being seen while streaming from Sunshine without having to uninstall ViGEmBus driver on host PC.
My use-case, Cyberpunk 2077 will see the virtual gamepad and always show it active and glitching control icons while streaming from Sunshine. Playing on the host PC without Sunshine running, the virtual gamepad is not detected and the icons show as Keyboard and Mouse normally.

**Comment by ReenigneArcher** [comment, marked as answer]
It sounds like you're connecting your gamepad directly to the host, instead of going through Moonlight. In Moonlight disable the option to always have a gamepad present.
